config/initializers/node-orm2.js

```diff
@@ -19,9 +19,6 @@ export default {
         adapter.db = db;
         let models = container.lookupAll('model');
         let adapterModels = mapValues(models, (Model) => {
-          if (Model.hasOwnProperty('abstract') && Model.abstract) {
-            return;
-          }
```

Comment:
I think the goal here was to avoid creating a node-orm2 model for ApplicationModel